src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,25 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={`${completed ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'}`}
```

Comment:
```suggestion
      className={`${completed ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'}`}
```
OMG)) Better separate it into few lines)

Reply:
Use `classnames` library for this.

Reply:
fixed